css/css-grid/animation/WEB_FEATURES.yml

```diff
@@ -0,0 +1,3 @@
+features:
+- name: grid-animation
```

Comment:
Is the name meant to reference a spec? If so this is testing part of the css-grid spec: https://drafts.csswg.org/css-grid/#track-sizing